### PR TITLE
feat: add datetime utils to package

### PR DESCRIPTION
**Pull request by kaisermann**
- Moves the `datetime` utils from CFTR to this package

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,6 +85,11 @@
     "typescript": "^4.4.4"
   },
   "dependencies": {
+    "date-fns": "2.x",
+    "luxon": "2.x",
     "murmurhash-js": "^1.0.0"
+  },
+  "peerDependencies": {
+    "@getcircuit/schema": "0.x"
```
Comment (gabrielreisn) [inline package.json:93]: why do we need it as a peerDep if it's already part of the bundle on the dependencies?

Comment (kaisermann) [reply package.json:93]: It's a dev and peer dep, not a dep. It's not bundled with the lib

Comment (gabrielreisn) [reply package.json:93]: oh, you're right. I think it's a sign to stop reading PR's on a cellphone 🙈

**Review by gabrielreisn — APPROVED** [review]

File: src/index.ts [hunk 2]
```diff
@@ -10,3 +10,4 @@ export * from './modules/regex'
 export * from './modules/sampleFn'
 export * from './modules/throttleFn'
 export * from './modules/time'
+export * from './modules/dateTime'
```
Comment (klzns) [inline src/index.ts:13]: I think we should move the time functions to the time module.

Comment (kaisermann) [reply src/index.ts:13]: done sensei

**Review by klzns — APPROVED** [review]

**Pull request merged**